plugins/workflow/message.go

```diff
@@ -181,6 +189,36 @@ func (cmd ExecuteActivity) ActivityParams(env bindings.WorkflowEnvironment) bind
 	return params
 }
 
+// ActivityParams maps activity command to activity params.
+//func (cmd ExecuteLocalActivity) ActivityParams(env bindings.WorkflowEnvironment) bindings.ExecuteLocalActivityOptions {
+//	params := bindings.ExecuteActivityParams{
+//		ExecuteActivityOptions: cmd.Options,
+//		ActivityType:           bindings.ActivityType{Name: cmd.Name},
+//		Input:                  cmd.rawPayload,
+//	}
+//
+//	if params.TaskQueueName == "" {
+//		params.TaskQueueName = env.WorkflowInfo().TaskQueueName
+//	}
+//
+//	return params
+//}
+
+// ActivityParams maps activity command to activity params.
```

Comment:
Remove commented code

Reply:
This snippet is currently blocked by SDK modification.